Header header
string cam_name
ObjectBox[] obj_boxes
